samples/test/automlTablesPredict.v1beta1.test.js

```diff
@@ -17,72 +18,113 @@
+      {stringValue: 'married'}, // MaritalStatus
+      {stringValue: 'secondary'}, // Education
+      {stringValue: 'no'}, // Default
+      {numberValue: 52}, // Balance
+      {stringValue: 'no'}, // Housing
+      {stringValue: 'no'}, // Loan
+      {stringValue: 'cellular'}, // Contact
+      {numberValue: 12}, // Day
+      {stringValue: 'aug'}, // Month
+      {numberValue: 96}, // Duration
+      {numberValue: 2}, //Campaign
+      {numberValue: -1}, // PDays
+      {numberValue: 0}, // Previous
+      {stringValue: 'unknown'}, // POutcome
+    ];
+
+    const payload = await samples.predict.main(
+      projectId,
+      region,
+      modelId,
+      inputs
+    );
+
+    assert(payload);
+    assert.ok(payload.hasOwnProperty('payload'));
   });
 
   it.skip(`should perform batch prediction using GCS as source and
     GCS as destination`, async () => {
     // Run batch prediction using GCS as source and GCS as destination
-    const output = exec(
-      `${cmdPredict} predict-using-gcs-source-and-gcs-dest "${modelId}"` +
-        ` "${gcsInputUri}" "${gcsOutputUriPrefix}"`
+    const operation = samples.predictGcs.main(
+      projectId,
+      region,
+      modelId,
+      gcsInputUri,
+      gcsOutputUriPrefix
     );
-    assert.match(output, /Operation name:/);
+    assert(operation);
   });
 
   it.skip(`should perform batch prediction using BQ as source and
     GCS as destination`, async () => {
     //  Run batch prediction using BQ as source and GCS as destination
-    const output = exec(
-      `${cmdPredict} predict-using-bq-source-and-gcs-dest "${modelId}"` +
-        ` "${bqInputUri}" "${gcsOutputUriPrefix}"`
+    const output = samples.predictBq.main(
+      projectId,
+      region,
+      modelId,
+      bqInputUri,
+      gcsOutputUriPrefix
     );
-    assert.match(output, /Operation name:/);
+    assert(output);
   });
 
   it.skip(`should perform batch prediction using GCS as source and
     BQ as destination`, async () => {
     // Run batch prediction using GCS as source and BQ as destination
-    const output = exec(
-      `${cmdPredict} predict-using-gcs-source-and-bq-dest "${modelId}"` +
-        ` "${gcsInputUri}" "${bqOutputUriPrefix}"`
+    const output = samples.predictBq.main(
+      projectId,
+      region,
+      modelId,
+      gcsInputUri,
+      bqOutputUriPrefix
     );
-    assert.match(output, /Operation name:/);
+
+    assert(output);
   });
 
   it.skip(`should perform batch prediction using BQ as source and
     BQ as destination`, async () => {
     // Run batch prediction using BQ as source and BQ as destination
-    const output = exec(
-      `${cmdPredict} predict-using-bq-source-and-bq-dest "${modelId}"` +
-        ` "${bqInputUri}" "${bqOutputUriPrefix}"`
+    const output = samples.predictBq.main(
+      projectId,
+      region,
+      modelId,
+      bqInputUri,
+      bqOutputUriPrefix
     );
-    assert.match(output, /Operation name:/);
```

Comment:
Why the move to `assert` vs `assert.match`?  It would be good to be just a little more careful about checking the output.

Reply:
They are in process. I will address them soon in another PR